File: Java/Basics To Java/src/A/A13.java
package A;

import java.util.Scanner;

public class A13 {

	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("zadej délku odvìsny1 (cm)");
		int odvesna1 = sc.nextInt();
		System.out.println("zadej délku odvìsny1 (cm)");
		int odvesna2 = sc.nextInt();
		float vypocet = (float) Math.sqrt(Math.pow(odvesna1,2)+ Math.pow(odvesna2,2));
		System.out.println("pøepona je dlouhá: " + vypocet + "cm");
	}
}

File: C#/Basic logic/A/A13.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cšarp_příklady
{
    internal class A13
    {
        static void Mainx(string[] args)
        {

            Console.WriteLine("Zadej délky odvěsen pravoúhlého trojuhelníka. (cm)");
            int odvesna1= int.Parse(Console.ReadLine());
            int odvesna2 = int.Parse(Console.ReadLine());
            double prepona = Math.Sqrt(Math.Pow(odvesna1,2)+Math.Pow(odvesna2,2));
            Console.WriteLine("prepona: " + prepona);
 
        }

    }
}
